migrations/20231128_jellyfish_migration/utils/jellyfish_data.go

```diff
@@ -0,0 +1,175 @@
+package utils
```

Comment:
What all are we doing here? Just adding the deduplicator name, version, and hash, right? Can we just replace this with a direct Mongo update?

Reply:
yeah I am beginning to wonder if it is event required

Reply:
after discussion with Jimmy and how he ran the test I don't think this is required any longer

Reply:
Why don't you keep this code in a separate saved feature branch, just in case.